solution-base/deps.yaml

```diff
@@ -8,8 +8,9 @@ mongodb-exporter:
   image: bitnami/mongodb-exporter
   tag: 0.11.2-debian-10-r410
 mongodb-sharded:
-  image: bitnami/mongodb-sharded
-  tag: 4.0.27-debian-9-r111
```

Comment:
What we may do is update (or build) against the latest "debian-9-r118" (or more?) bitnami minideb.
Not sure it will solve much CVEs though